E45F01FFFEEFD308.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "E45F01FFFEEFD308",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
```

Comment:
This URL should be updated. This is the old v2 routers, please use the v3 ones and then we can merge.